### PR TITLE
Build package deb rpm

### PR DESCRIPTION
**Pull request by davide-baldo**
- Created ZAL package
- Removed old builder/scripts

### CODE REVIEW

**Review by mzanellazextras — COMMENTED** [review]
How do we want to manage the packaging of the same ZAL version for the different Carbonio versions?

e.g. suppose we have 2 Carbonio versions, say 25.1 and 26.7. We have to release the version of ZAL, say 4.5.2 for both of them. Is the previous one a feasible scenario? Do we want to delegate to the one that wants to install ZAL the responsibility to install to correct version?

**Review by Polpetta — APPROVED** [review]
lgtm :+1:

File: Jenkinsfile [hunk 1]
```diff
@@ -12,20 +15,22 @@ pipeline {
...
+                checkout scm
                 withCredentials([file(credentialsId: 'jenkins-maven-settings.xml', variable: 'SETTINGS_PATH')]) {
                     sh "cp ${SETTINGS_PATH} settings-jenkins.xml"
                 }
             }
         }
         stage('Build') {
             steps {
-                sh 'mvn --settings settings-jenkins.xml -Dzimbra.version=8.8.15  package'
+                sh 'mvn -B --settings settings-jenkins.xml -Dzimbra.version=8.8.15  package'
```
[the remark this answers is not in the document]
Comment (davide-baldo) [reply Jenkinsfile:31]: Done

**Comment by davide-baldo** [comment]
> How do we want to manage the packaging of the same ZAL version for the different Carbonio versions?
> 
> e.g. suppose we have 2 Carbonio versions, say 25.1 and 26.7. We have to release the version of ZAL, say 4.5.2 for both of them. Is the previous one a feasible scenario? Do we want to delegate to the one that wants to install ZAL the responsibility to install to correct version?

We can specify the minimum and maximum version in the dependent package, see other PRs

**Review by mzanellazextras — APPROVED** [review]

**Pull request merged**